docs/mixin/alerts/alerts.yaml

```
@@ -173,7 +173,7 @@ groups:
       severity: warning
     annotations:
       summary: High cache eviction in Promscale
-      description: "Promscale {{ $labels.name }} is evicting at {{ $value }} entries a second."
+      description: "Promscale {{ $labels.name }} is evicting at {{ $value | humanizePercentage }} entries a second."
```

```suggestion
      description: "Promscale {{ $labels.name }} is evicting at {{ $value }} entries a second."
```
value is not in percent but in seconds.